addons/longer-wiwo/userscript.js

```diff
@@ -21,5 +21,5 @@ export default async function ({ msg }) {
     left.hide();
   });
 
-  status[0].maxLength = 255; // disallow more than 255 chars
+  status[0]?.maxLength = 255; // disallow more than 255 chars
```

Comment:
Please do it the old way for better browser support?

Reply:
[Chromium and Firefox have both supported it for a year...](https://caniuse.com/mdn-javascript_operators_optional_chaining)

Reply:
Oh wow I thought it was newer. Still, you could just have used an if here just fine :P